selfdrive/test/process_replay/process_replay.py

```diff
@@ -18,11 +21,12 @@ def tqdm(x):
 from common.params import Params
 from cereal.services import service_list
 from collections import namedtuple
+from common.basedir import BASEDIR
 
 # Numpy gives different results based on CPU features after version 19
 NUMPY_TOLERANCE = 1e-7
 
-ProcessConfig = namedtuple('ProcessConfig', ['proc_name', 'pub_sub', 'ignore', 'init_callback', 'should_recv_callback', 'tolerance'])
+ProcessConfig = namedtuple('ProcessConfig', ['lang', 'proc_name', 'pub_sub', 'ignore', 'command', 'path', 'init_callback', 'should_recv_callback', 'tolerance'])
```

Comment:
we don't need any of this extra stuff, it's already in manager

Reply:
https://github.com/commaai/openpilot/blob/master/selfdrive/manager.py#L288

Reply:
I deal with sockets differently depending on the language

Reply:
you can borrow the same logic manager uses instead of adding this - https://github.com/commaai/openpilot/blob/master/selfdrive/manager.py#L292

Reply:
Hm ok I can try refactoring

Reply:
Will keep lang, because the parsing of received packages is quite different, but will remove command and path